.travis.yml

```diff
@@ -1,6 +1,17 @@
 language: node_js
-node_js:
-  - "6"
```

Comment:
Since you are removing support for Node 6 and 8, this commit message needs to have a `BREAKING CHANGE` in it.